### PR TITLE
CNV-42087: Improve sysprep section when the answer file is added

### PR DESCRIPTION
**Pull request by avivtur**
## 🎥 Demo

Before:
sysprep-display-b4

After:

sysprep-display2-gutter

### CODE REVIEW

**Comment by avivtur** [comment]
/hold 
Until #1998 is merged

**Review by metalice — APPROVED** [review]

**Pull request merged**
